src/main/java/mate/academy/spring/controller/ShoppingCartController.java

```diff
@@ -0,0 +1,42 @@
+package mate.academy.spring.controller;
+
+import mate.academy.spring.mapper.DtoResponseMapper;
+import mate.academy.spring.model.ShoppingCart;
+import mate.academy.spring.model.dto.response.ShoppingCartResponseDto;
+import mate.academy.spring.service.MovieSessionService;
+import mate.academy.spring.service.ShoppingCartService;
+import mate.academy.spring.service.UserService;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+
+public class ShoppingCartController {
+    private final DtoResponseMapper<ShoppingCartResponseDto, ShoppingCart>
+            shoppingCartDtoResponseMapper;
+    private final ShoppingCartService shoppingCartService;
+    private final MovieSessionService movieSessionService;
+    private final UserService userService;
+
+    public ShoppingCartController(
+            ShoppingCartService shoppingCartService,
+            DtoResponseMapper<ShoppingCartResponseDto, ShoppingCart> shoppingCartDtoResponseMapper,
+            MovieSessionService movieSessionService, UserService userService) {
+        this.shoppingCartService = shoppingCartService;
+        this.shoppingCartDtoResponseMapper = shoppingCartDtoResponseMapper;
+        this.movieSessionService = movieSessionService;
+        this.userService = userService;
+    }
+
+    @PutMapping
+    public void addMovieSession(@RequestParam Long userId, @RequestParam Long movieSessionId) {
```

Comment:
And here also need an annotation
```suggestion
    @PutMapping("/movie-sessions")
    public void addMovieSession(@RequestParam Long userId, @RequestParam Long movieSessionId) {
```